pytest_perf/plugin.py

```diff
@@ -36,9 +62,18 @@ def pytest_sessionfinish():
 
 
 class File(pytest.File):
+    def __init__(self, fspath, parent, target=".", baseline=None):
+        super().__init__(fspath, parent)
+        self.target = target
+        self.baseline = baseline
```

Comment:
I'd like to avoid this plumbing (and repeated defaults) if possible. I wonder if pytest exposes the config such that it is available when the runner is constructed.